### PR TITLE
import academic data

### PR DESCRIPTION
**Pull request by shreyaswikriti**
Imported contents from [teaching-ocaml page](https://github.com/ocaml/ocaml.org/blob/master/site/learn/teaching-ocaml.md).

- Added logo images of the institutions for the new academic page.
- imported institution name, courses, resources, location, etc

### CODE REVIEW

**Comment by patricoferris** [comment]
Thanks for the PR @shreyaswikriti, this all looks really good from a quick glance. The CI is failing because the code has not been formatted (although to be fair we don't say anywhere that it must be formatted). Could you reformat the code, commit it and push to this branch again? 

To format the code you will need to install ocamlformat `opam install ocamlformat.0.15.0` and then from the root of the project you can run `dune build @fmt --auto` this will run the formatter and automatically replace the unformatted lines of code with formatted ones. This should fix the failing step in the CI and then we can get to the lint checks to make sure the data is all working nicely :))

File: data/academic_institution/aix.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+url: "https://www.univ-amu.fr/en"
+logo: academic_institution/aix.jpg
+continent: Europe
+courses:
+    - name: Functional Programming   
+location:
+     lat: 43.3048
+     long: 5.3774
+---
+s
```
Comment (tmattio) [inline data/academic_institution/aix.md]: typo?

**Comment by tmattio** [comment]
This is great, thanks @shreyaswikriti!

I don't see any markdown in the document, only the metadata. Is there a reason to add the data as markdown documents, as opposed to a standalone Yaml file?

**Comment by patricoferris** [comment]
I suggested to do the markdown format, the idea is that I would quite like institutions to add comments about OCaml and how they use it, we just don't have this information yet (similar to what the markdown is used for in the industrial users section).

File: data/academic_institution/aix.md (hunk 1 above)
Comment (shreyaswikriti) [reply data/academic_institution/aix.md]: Thanks for pointing this out. I'll check for typos if any.

> typo?

**Comment by patricoferris** [comment]
Awesome all tests are green, thanks @shreyaswikriti! 🎉

**Pull request merged**